### DISCUSSION TITLE
How to change the currency

### DISCUSSION THREAD

**Discussion by abisha-janet** [bitsandbots/inventory#12] [Q&A]
Hi there,

I have a few questions,
* How to change the currency from USD to OMR ?
* How can I search by products name instead of category in products.php?

**Comment by bitsandbots** [comment]
The format currency setting that needs to be changed is found at /inventory/includes/load.php after the list of included files.

The products/product_search.php will provide the ability to search by name/sku/description

**Reply by abisha-janet** [reply]
Thank you for update me.
Currency I have changed.
But search, it's not in the product_search.php; need to change search option by products name instead of product category in products page (products.php).
If I change the code as in product_search.php, It didn't showup the results.
